Update docs of REPLs (#712)
teloxide/teloxide: src/dispatching/repls/stopping.md

> To stop repl, simply press `Ctrl`+`C` in the terminal where you run the program.
> Note that gracefully stopping can take some time (we plan to improve this, see [#711]).

Reviewer: ```suggestion
To stop a REPL, simply press `Ctrl`+`C` in the terminal where you run the program.
Note that graceful shutdown may take some time (we plan to improve this, see [#711]).
```
